packages/core/test/acceptance/handlers.acceptance.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/core
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  Application,
+  Handler,
+  httpHandler,
+} from '../..';
+import {expect, sinon} from '@loopback/testlab';
+
+describe('http handler', () => {
+  let app: Application;
+  before(givenApp);
+
+  it('returns a Handler', () => {
+    const handler: Handler = httpHandler(app);
+    expect(handler).to.be.type('function');
+  });
+
+  context('Handler', () => {
+    // tslint:disable-next-line
+    let stub: any;
```

Comment:
we should be exporting more `sinon` types from `testlab`, so that this line can become

```ts
let stub: SinonStub;
```